RecoTracker/LSTCore/src/ModuleMethods.h

```diff
@@ -356,12 +361,21 @@ namespace lst {
       if (detId != 1) {
         host_partnerModuleIndices[index] =
             mmd.detIdToIndex[parsePartnerModuleId(detId, host_isLower[index], host_isInverted[index])];
+        auto const partnerIdx = host_partnerModuleIndices[index];
         //add drdz and slope importing stuff here!
         if (host_drdzs[index] == 0) {
-          host_drdzs[index] = host_drdzs[host_partnerModuleIndices[index]];
+          host_drdzs[index] = host_drdzs[partnerIdx];
         }
         if (host_dxdys[index] == 0) {
-          host_dxdys[index] = host_dxdys[host_partnerModuleIndices[index]];
+          host_dxdys[index] = host_dxdys[partnerIdx];
+        }
+        if (host_isLower[index]) {
+          if (host_subdets[index] == Barrel) {
+            host_isGloballyInner[index] = host_r[index] < host_r[partnerIdx];
```

Comment:
Is this guaranteed to work in the case of tilted modules, where the strip module (based on its center) may register closer than a pixel one r-wise? I think this is the reason I had done something more complicated in the `LSTOutputConverter`:
https://github.com/cms-sw/cmssw/blob/640e69c0fc493e6641ce99732a308b638a0e55fc/RecoTracker/LST/plugins/LSTOutputConverter.cc#L197-L201

Reply:
Did you actually see a difference?
They all should be rectangular (with no extra lateral offset between the sensors) and tangent to the rphi direction at the center: the rz projection should be a line of the same length for both sensors. So, r should be enough.

Since something doesn't work, chances are that this part is a problem.

Reply:
I think there was a difference, but it's been a while ..and I couldn't find any plots to support it...